### PR TITLE
Adds gisf to deployment flow

### PR DESCRIPTION
**Pull request by jaycee**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6738/

**Comment by jaycee** [comment]
QA:

* Run `make run`; kick off guiproxy with `-config 'gisf:true, jujuEnvUUID:""'`
* Go to /new
* Search for 'jcsackett' in the store--click on jcsackett/mysql and add to canvas.
* Open the dev tools and find the usso-login-link file. Search for the call to sendPost inside loginToControllers in the USSOLoginLink component and add a breakpoint here.
* Go through deployment flow, using the login or sign up button. The breakpoint should be hit. Remove the breakpoint and allow execution to continue.
* Finish DF.

File: jujugui/static/gui/src/app/app.js [hunk 1]
```diff
@@ -1273,6 +1273,10 @@ YUI.add('juju-gui', function(Y) {
       };
       const loginToController = controllerAPI.loginWithMacaroon.bind(
         controllerAPI, this.bakeryFactory.get('juju'));
+      const getDischargeToken = function() {
+        return window.localStorage.getItem('discharge-token', null);
```
Comment (hatched) [inline jujugui/static/gui/src/app/app.js]: I don't think there is a second argument for `getItem` - is this required?

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6740/

**Comment by hatched** [comment]
Thanks @jaycee QA OK
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6741/

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6742/

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge/2110

**Pull request merged**